.travis.yml

```diff
@@ -6,7 +6,7 @@ env:
     - SITL_RATE=200
 
 python:
-  - 2.7.13
+  - 2.7
```

Comment:
This change isn't motivated by your commit comment.  Why are we moving from the more-specific version?  Because we can?